quickstarts/library-process/README.md

```diff
@@ -1,8 +1,8 @@
 ## IBM Business Automation Manager Open Editions Kie Server Quickstart
 
-Note that, this is the development branch, the target images might not be available here, instead you can look at the [released branch](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.13.x-blue/quickstarts/library-process)
+This quickstart is intended to be used with the [IBM BAMOE Kie Server](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/8.0.3.GA/kieserver) image.
 
-This quickstart is intend to be used with the [IBM BAMOE Kie Server](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.13.x-blue/kieserver) image.
+## How to use it?
```

Comment:
Seems the `# How to use it?` is duplicated as it was already in the README before.

Reply:
@spolti I think this one here is relevant too. The links I think can be ignored as you mentioned in the chat.